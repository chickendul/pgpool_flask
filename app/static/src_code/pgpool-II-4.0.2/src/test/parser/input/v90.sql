version https://git-lfs.github.com/spec/v1
oid sha256:ec1016fd3c0082fabfa247e115e66d51719c3b6baf933fb6816d1ceda1486469
size 1344
