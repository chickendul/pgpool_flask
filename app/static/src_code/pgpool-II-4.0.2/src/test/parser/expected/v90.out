version https://git-lfs.github.com/spec/v1
oid sha256:b61fd7cdcf93bae723cafbf331c725803c683bc57c10945edaa18005a5348994
size 1184
